src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -969,6 +969,37 @@ public void insertIntoDatasetTable(
     bigQueryProject.query(sqlTemplate.render());
   }
 
+  private static final String insertIntoMetadataTableTemplate =
+      "INSERT INTO `<project>.<dataset>.<metadataTable>` (<rowIdColumn>, <ingestByColumn>, "
+          + "<ingestTimeColumn>, <loadTagColumn>) "
+          + "SELECT <rowIdColumn>, '<ingestEmail>' AS <ingestByColumn>, "
+          + "CURRENT_TIMESTAMP() AS <ingestTimeColumn>, '<loadTag>' AS <loadTagColumn> "
```

Comment:
Does setting this to CURRENT_TIMEPSTAMP() mean that all of the rows in this load have the same timestamp? Or is it slightly off between the different between rows?
Just curious, I think either is fine.

Reply:
I think we'd want the same timestamp for all rows.

Reply:
I think this would use the same timestamp for everything that's returned by the select statement but I can double check that behavior

Reply:
So this just gets one timestamp value and inserts that into the metadata table (I added an assert statement in the test for this too).